pkg/utils/diff.go

```diff
@@ -0,0 +1,78 @@
+package utils
```

Comment:
Please put under analyzer package and add test for `GetSuggestedFix`.

Reply:
sure

Reply:
done